components/Home.go

```diff
@@ -140,6 +148,11 @@ func focusTab(tab *Tab) {
 			App.SetFocus(table)
 		}
 
+		if tab.Name == "Editor" {
```

Comment:
Is there no constant for this? Maybe it's worth adding it and use it where needed